### ISSUE TITLE
Some refused on predict with SVM non linear kernel

### ISSUE THREAD

**Issue by trangtv57** [nfmcclure/tensorflow_cookbook#99]
Hi  nfmcclure,
Sorry for ask again about predict points in SVM. But before I can ask something I dont understand, you have closed my topi. I have some confused when apply your example SVM non linear for my case. I already train SVM non linear with gaussian kernel for iris dataset. Everything is ok, train and predict with all data. But when I try to predict some point in dataset, all y label is 0 not 1 or -1. 
My code here:
### I create my point is 1 point in dataset.
`
predict_grid = np.float32(np.array([[5.0999999, 3.5, 1.39999998, 0.2]]))
rA = tf.reshape(tf.reduce_sum(tf.square(x_data), 1), [-1, 1])
rB = tf.reshape(tf.reduce_sum(tf.square(predict_grid), 1), [-1, 1])
pred_sq_dist = tf.add(tf.subtract(rA, tf.multiply(2., tf.matmul(x_data, tf.transpose(predict_grid)))),
                      tf.transpose(rB))
pred_kernel = tf.exp(tf.multiply(gamma, tf.abs(pred_sq_dist)))
`

### I have some confuse here, We need provide ( y_target  mean true label with variable of svm) to predict point, so if b_tf have batch size < size of sample y, so how I choose  y target ??
`
prediction_output = tf.matmul(tf.multiply(tf.transpose(y_target), b_tf), pred_kernel)
prediction = tf.sign(prediction_output - tf.reduce_mean(prediction_output))
`
#### all value prediction is 0 
`
print(prediction.eval())
`

**Comment by nfmcclure** [comment]
Hi @trangtv57 , thanks for the bug report. Sorry about the late reply, I'm just now getting around to traiging the issues in preparation for a code v2 of the book.

I should be getting around to this in the next few months.  Thanks again!

**Comment by nfmcclure** [comment]
This should be fixed now. Try repulling the updated code. Thanks!
